core/server/master/src/test/java/alluxio/master/file/FileSystemMasterFsOptsTest.java

```diff
@@ -261,279 +261,167 @@ public void deleteDirRecursiveWithReadOnlyCheck() throws Exception {
         DeleteContext.mergeFrom(DeletePOptions.newBuilder().setRecursive(true)));
   }
 
-  @Test
-  public void deleteDirRecursiveWithInsufficientPermissions() throws Exception {
-    // userA has permissions to delete directory but not one of the nested files
-    createFileWithSingleBlock(NESTED_FILE_URI);
-    createFileWithSingleBlock(NESTED_FILE2_URI);
-    mFileSystemMaster.setAttribute(NESTED_URI, SetAttributeContext
-        .mergeFrom(SetAttributePOptions.newBuilder().setMode(new Mode((short) 0777).toProto())));
-    mFileSystemMaster.setAttribute(NESTED_FILE_URI, SetAttributeContext
-        .mergeFrom(SetAttributePOptions.newBuilder().setMode(new Mode((short) 0700).toProto())));
-    mFileSystemMaster.setAttribute(NESTED_FILE2_URI, SetAttributeContext
-        .mergeFrom(SetAttributePOptions.newBuilder().setMode(new Mode((short) 0777).toProto())));
-    try (AuthenticatedClientUserResource userA = new AuthenticatedClientUserResource("userA",
-        Configuration.global())) {
-      mFileSystemMaster.delete(NESTED_URI,
-          DeleteContext.mergeFrom(DeletePOptions.newBuilder().setRecursive(true)));
-      fail("Deleting a directory w/ insufficient permission on child should fail");
-    } catch (FailedPreconditionException e) {
-      assertTrue(e.getMessage().contains("/nested/test/file (Permission denied"));
-      assertTrue(e.getMessage().contains("/nested/test (Directory not empty)"));
-    }
-    // Then the nested file and the dir will be left
-    assertNotEquals(IdUtils.INVALID_FILE_ID, mFileSystemMaster.getFileId(NESTED_URI));
-    assertNotEquals(IdUtils.INVALID_FILE_ID, mFileSystemMaster.getFileId(NESTED_FILE_URI));
```

Comment:
why are these UTs removed? Would there any replacing ones?

Reply:
This case is used to test the effect of deleting files with different permissions, but this pr no longer judges the permissions of the file itself.